railties/lib/rails/templates/rails/mailers/email.html.erb

```diff
@@ -128,16 +167,31 @@
     <dt>EML File:</dt>
     <dd><%= link_to "Download", action: :download %></dd>
   </dl>
+
+  <div id="controlView">
+    <a class="active controlItem" href="#" onclick="swithToDesktop(this); return false">
+      <img title="Desktop View (600px)" height="32" src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/PjwhRE9DVFlQRSBzdmcgIFBVQkxJQyAnLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4nICAnaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkJz48c3ZnIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDQ4IDQ4IiBoZWlnaHQ9IjQ4cHgiIHZlcnNpb249IjEuMSIgdmlld0JveD0iMCAwIDQ4IDQ4IiB3aWR0aD0iNDhweCIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+PGcgaWQ9IkV4cGFuZGVkIj48Zz48Zz48cGF0aCBkPSJNNDQsNDJINGMtMi4yMDYsMC00LTEuODc0LTQtNC4xNzdWOC4xNzdDMCw1Ljg3NCwxLjc5NCw0LDQsNGg0MGMyLjIwNiwwLDQsMS44NzQsNCw0LjE3N3YyOS42NDYgICAgIEM0OCw0MC4xMjYsNDYuMjA2LDQyLDQ0LDQyeiBNNCw2QzIuODk3LDYsMiw2Ljk3NywyLDguMTc3djI5LjY0NkMyLDM5LjAyMywyLjg5Nyw0MCw0LDQwaDQwYzEuMTAzLDAsMi0wLjk3NywyLTIuMTc3VjguMTc3ICAgICBDNDYsNi45NzcsNDUuMTAzLDYsNDQsNkg0eiIvPjwvZz48Zz48cGF0aCBkPSJNMzgsNDZIMTBjLTAuNTUyLDAtMS0wLjQ0OC0xLTFzMC40NDgtMSwxLTFoMjhjMC41NTIsMCwxLDAuNDQ4LDEsMVMzOC41NTIsNDYsMzgsNDZ6Ii8+PC9nPjxnPjxwYXRoIGQ9Ik0yNCw0NmMtMC41NTIsMC0xLTAuNDQ4LTEtMXYtNGMwLTAuNTUyLDAuNDQ4LTEsMS0xczEsMC40NDgsMSwxdjRDMjUsNDUuNTUyLDI0LjU1Miw0NiwyNCw0NnoiLz48L2c+PGc+PGNpcmNsZSBjeD0iMjQiIGN5PSIzNyIgcj0iMiIvPjwvZz48Zz48cmVjdCBoZWlnaHQ9IjIiIHdpZHRoPSI0NiIgeD0iMSIgeT0iMzIiLz48L2c+PC9nPjwvZz48L3N2Zz4="/>
```

Comment:
I'm also interested on the source of these icons. Where do they come from and what license do they have?